sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -53,68 +50,40 @@ add the direct dependency to your project as follows.
+- An [Azure Subscription][azure_subscription].
+- An existing [Azure Key Vault][azure_keyvault]. If you need to create a key vault, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-key-vault-name>` and `<your-resource-group-name>` with your own, unique names:
 
 ```bash
-az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+az keyvault create --name "<your-key-vault-name>" --resource-group "<your-resource-group-name>"
 ```
 
 ### Authenticate the client
-In order to interact with the Azure Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the `DefaultAzureCredential` examples shown in this document.
-
-The `DefaultAzureCredential` way of authentication by providing client secret credentials is being used in this getting started section, but you can find more ways to authenticate with [azure-identity][azure_identity].
-
-#### Create/Get credentials
-To create/get client secret credentials you can use the [Azure Portal][azure_create_application_in_portal], [Azure CLI][azure_keyvault_cli_full] or [Azure Cloud Shell][azure_cloud_shell]
-
-Here is an [Azure Cloud Shell][azure_cloud_shell] snippet below to
-
- * Create a service principal and configure its access to Azure resources:
-
-```bash
-az ad sp create-for-rbac -n <your-application-name> --skip-assignment
-```
-
-Output:
-
-```json
-{
-    "appId": "generated-app-ID",
-    "displayName": "dummy-app-name",
-    "name": "http://dummy-app-name",
-    "password": "random-password",
-    "tenant": "tenant-ID"
-}
-```
+In order to interact with the Azure Key Vault service, you will need to create an instance of the [`CertificateClient`](#create-certificate-client) class. You need a **vault url** and **[a managed identity][managed_identity]** to instantiate a client object using the `DefaultAzureCredential` examples shown in this document.
 
-* Use the above returned credentials information to set the **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Bash:
+The `DefaultAzureCredential` way of authentication with a system-assigned managed identity is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
 
-```bash
-export AZURE_CLIENT_ID="generated-app-ID"
-export AZURE_CLIENT_SECRET="random-password"
-export AZURE_TENANT_ID="tenant-ID"
-```
+#### Give a managed identity access to your key vault
+To give a managed identity access to your key vault you can use the [Azure Portal][azure_portal_managed_identity], [Azure CLI][azure_cli_managed_identity] or [Azure Cloud Shell][azure_cloud_shell].
```

Comment:
Now that I'm actually seeing this, I have some concerns. This works great for S2S communications, but not for developers. Should we also mention for dev-time log into the Azure CLI, VS, etc?

Reply:
I would lean on linking to identity readme here - to have a single point of information here.

Reply:
Ended up removing this whole section.